vspace/src/main/java/edu/asu/diging/vspace/web/staff/DefaultImageApiController.java

```diff
@@ -24,6 +24,7 @@ public class DefaultImageApiController {
     public static final String API_DEFAULT_MODULE_IMAGE_PATH = "/api/image/default/link/module/";
     public static final String API_DEFAULT_EXTERNAL_IMAGE_PATH = "/api/image/default/link/external/";
     public static final String API_DEFAULT_SPACE_IMAGE_STATUS = "/api/exhibition/default/link/image/status/";
+    public static final String API_DEFAULT_IMAGE_DISABLE_STATUS = "/api/exhibition/default/link/image/disableStatus/";
```

Comment:
it's the status of the disabled flag, right? So throughout this file it needs to say "disabled" not "disable". Otherwise it constantly raises the question if this is an action to disable the status.